content/guides/basic-setup/index.md

```diff
@@ -13,48 +13,55 @@ If you require further help, please visit our forums or ask in our Discord's hel
 
 ## How to dump your PS2 BIOS
 
-Dumping your PS2 BIOS requires a mod to your PS2 to allow arbitrary code execution.
+Dumping your PS2 BIOS is conceptually a two-step process:
+1. "Hack" your PS2 to allow arbitrary code execution
+2. Now you can run a "BIOS dumper" utility program on your PS2 that reads its BIOS and writes it to a USB drive.
 
-**Popular options:**
+There is a generally useful program, uLaunchELF, that lets you browse memory cards, DVDs, and USB drives connected to a PS and run programs from them. So in most of the approaches below, you "hack" your PS2 so that it runs uLaunchELF, and uLaunchELF in turn it lets you run the BIOS dumper utility from a USB drive.
+
+### Popular "hack" options
 
 1. FreeMcBoot Memory Card
-    - Works for all but the newest (9xxxx serial number with a date code larger than 8B) slim PS2s. Can be found online for ~10 USD. Minimal effort.
+    - Works for all but the newest (9xxxx serial number with a date code larger than 8B) slim PS2s. Can be found online for ~10 USD. Minimal effort. This boots into the uLaunchELF program.
 2. FreeDVDBoot
     - Works for many slim models, and some phat models. Slight effort, but free.
     - You will require a blank DVD for this method to work!
-      - It is recommended to use a **DVD-R** disc that supports atleast 4x speed, burned at 4x speed.
+    - This boots inot the uLaunchELF program
 3. Disc swap exploits
     - Technical in nature, involves hardware tampering. Guides can be found quickly by Googling.
 4. Mod chips
     - Extremely technical, requires soldering skills. DO NOT ATTEMPT unless you are an electronics pro.
 
-### Downloading the dumper utility
+### Downloading the BIOS dumper utility
 
 Hosted by the PCSX2 Project:
 - [Binary Version](https://github.com/PCSX2/tools/releases/download/bios-dumper%2Fv2/PS2dumperV2_bin.7z) (Recommended)
+  - After downloading, extract the files to a USB flash drive.
+    - Your mileage may vary here. All PS2 models can read and write to USB flash drives formatted with a FAT32 file system, but not the larger capacity exFAT format. Some people report USB 3.0 drives being usable while others claim they are not.  For this reason it appears to be more dependent on the drive rather than the USB version so we cannot provide an exhaustive list for success.
```

Comment:
Keep the information concise, mentioning exFAT is pointless here.
As an example

> All PS2 models can read and write to USB flash drives formatted with a FAT32 file system. Some USB 3.0 drives were reported as incompatible...

Edit: I see this was just moved, and isn't your writing now.